conf/ocsci/default_config.yaml

```diff
@@ -22,8 +22,8 @@ DEFAULTS:
   ROOK_CLUSTER_NAMESPACE: "openshift-storage"
   KUBECONFIG_LOCATION: "auth/kubeconfig"  # relative from cluster_dir
   API_VERSION: "v1"
-  CEPH_IMAGE: "quay.io/rhceph-dev/rhceph-4.0-rhel-8"
-  ROOK_IMAGE: "quay.io/rhceph-dev/rook"
+  CEPH_IMAGE: "ceph/ceph:v14.2.0-20190410"
```

Comment:
can we use the consts in defaults.py?
In case we will need to change the images, we can forget one place

Reply:
Even we have the same in defaults.py and we are going to deprecate defaults.py once we switched to pytest completely.

Reply:
I am not sure we are going to deprecate defaults.py, this was not discussed?

Reply:
I am going to take this up later in group!

Reply:
> I am not sure we are going to deprecate defaults.py, this was not discussed?

From one of the Petr comments 

```
Please update images in default config:
conf/ocsci/default_config.yaml

This file is basically overwriting everything in DEFAULTS section which is with capital letters to ocs/defaults.py file. This ocs/defaults.py will be depricated once old runner deleted and we should use the yaml configuration which will be exposed here to defaults. It's done here.

If you didn't use the conf/ocs_basic_install.yml file when you verified this, I think that images from conf/ocsci/default_config.yaml were used anyway.

Maybe they already updated images
```